scripts/development/build-binaries.sh

```
@@ -6,7 +6,11 @@ mkdir -p .bin
 
 # Read version from repo root .version (trim whitespace), fallback to "dev"
 VERSION=$(cat ../.version | sed 's/^\s*\|\s*$//g')
-LDFLAGS="-w -s -buildid=${VERSION}"
+REVISION=$(cat ../.revision 2>/dev/null || git rev-parse --short HEAD 2>/dev/null || echo "unknown")
+
+LDFLAGS="-w -s -buildid=${VERSION} \
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)